app/controllers/admin/items_controller.rb

```diff
@@ -9,7 +9,10 @@ def index
 
   def create
     @item = Item.new(item_params)
+
     if @item.save
+      category_collection = params[:item][:categories].reject {|x| x.empty?}
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Pass `&:empty?` as an argument to `reject` instead of a block.